src/org/zaproxy/zap/extension/ascanrulesAlpha/files/json/config.json

```diff
@@ -0,0 +1,10 @@
+{
+"files":[
+{
+"name":"file1.txt"
+},
+{
+"name":"testfile2.php"
+}
+]
+}
```

Comment:
If this is going to be json it may as well be well formatted. Also this should likely draw on the content from this original ticket, not just be some random "stuff".

Reply:
I'm not clear about what needs to be done here. Could you please elaborate? And do I need to leave sample entries there?

Reply:
Have a look at the ticket. Review the content it links to, review the project it discusses and the way it functions. Do something similar here (w/ attribution etc.)

Reply:
In addition to having the files/paths to test the file should also handle follow-on content checks. As you can see in the referenced article etc the tool not only looks for the URLs but in many instances also check the response content in order to reduce false positives.